src/tasks/generateSignatures.ts

```diff
@@ -16,6 +17,12 @@ const generateSignatures: () => void = () => {
       "fileWithAddress",
       "File with comma separated addresses that should be allow-listed",
     )
+    .addOptionalParam(
+      "postToApi",
+      "File with comma separated addresses that should be allow-listed",
+      "false",
```

Comment:
I think this should be boolean:
```suggestion
      false,
```